azure-pipelines.yml

```diff
@@ -50,7 +50,7 @@ steps:
 - bash: |
     source activate $ENV_NAME
     # Install package using pip
-    pip install . --no-deps
+    conda install .
```

Comment:
What is the reason for this change?

Reply:
In the azure-pipelines.yml it says that it should test for using conda and only conda prompts.
It used to use pip, which does not use conda. That's why I changed it. But I can change it back as it is not part of this PR.

Reply:
> In the azure-pipelines.yml it says that it should test for using conda and only conda prompts. It used to use pip, which does not use conda. That's why I changed it. But I can change it back as it is not part of this PR.

This refers to dependencies/environment. Generally it is good practice not to mix different things in PR. If you think, this needs to be changed, it should be in a separate PR.

Reply:
I agree. I have now reverted this change. Ready for review once more.